test/performance/shmem_perf_suite/bw_common.h

```diff
@@ -198,25 +222,62 @@ void static command_line_arg_check(int argc, char *argv[],
             break;
         case 'v':
             metric_info->validate = true;
+            if(metric_info->target_data) error = true;
             break;
         case 'w':
             metric_info->window_size = strtoul(optarg, (char **)NULL, 0);
+            if(metric_info->target_data) error = true;
+            break;
+        case 't':
+            metric_info->target_data = true;
+            metric_info->window_size = 1;
+            if(metric_info->validate) error = true;
+            break;
+        case 'r':
+            metric_info->sztarget = strtoul(optarg, (char **)NULL, 0);
+            if(metric_info->sztarget > metric_info->midpt ||
+                metric_info->szinitiator < metric_info->midpt ||
+                !metric_info->target_data) error = true;
+            break;
+        case 'l':
+            metric_info->szinitiator = strtoul(optarg, (char **)NULL, 0);
+            if(metric_info->szinitiator > metric_info->midpt ||
+                metric_info->sztarget < metric_info->midpt ||
+                !metric_info->target_data) error = true;
             break;
         default:
             error = true;
             break;
         }
     }
 
+    /* filling in 8/4KB chunks into array alloc'd to max_len */
+    if(metric_info->target_data) {
+        metric_info->start_len = TARGET_SZ_MIN;
+        if((metric_info->max_len <
+            ((metric_info->trials + metric_info->warmup) * TARGET_SZ_MIN)) ||
+            (metric_info->max_len <
+            ((metric_info->trials + metric_info->warmup) * TARGET_SZ_MAX))) {
+                error = true;
+            }
+    }
+
     if (error) {
         if (metric_info->my_node == 0) {
             fprintf(stderr, "Usage: \n[-s start_length] [-e end_length] "\
```

Comment:
The backslashes at the end of these lines are not necessary.  C will paste adjacent string literals together, even when they are separated by line breaks.